specification/validated-relationships-service-api.yaml

```diff
@@ -537,9 +536,8 @@ paths:
         | A single proxy relationship between a mother and child                            | `performer:identifier`=`9000000019` and/or `patient:identifier`=`9000000009`                                                                     | HTTP Status 200 Bundle containing a single proxy relationship                                                     |
         | A single proxy relationship between a mother and child including details          | `performer:identifier`=`9000000019` and/or `patient:identifier`=`9000000009` and `_include` = `Consent:patient` and `_include` = `Consent:proxy` | HTTP Status 200 Bundle containing a single proxy relationship including performer and patient details             |
-        | Invalid status parameter                                                          | `performer:identifier`=`9000000019` and/or `patient:identifier`=`9000000009` and `status` = `test`                                               | HTTP Status 422 and INVALID_VALUE error response                                                                  |
-        | Invalid include parameter                                                         | `performer:identifier`=`9000000019` and/or `patient:identifier`=`9000000009` and `_include` = `test`                                             | HTTP Status 422 and INVALID_PARAMETER error response                                                                  |
-        | Missing identifier                                                                | `patient:identifier`=`9000000009`                                                                                                                | HTTP Status 400 and MISSING_IDENTIFIER_VALUE error response                                                       |
+        | Invalid status parameter                                                          | `performer:identifier`=`9000000019` and/or `patient:identifier`=`9000000009` and `status` = `test`                                               | HTTP Status 422 and INVALID_PARAMETER error response                                                                  |
+        | Invalid include parameter                                                         | `performer:identifier`=`9000000019` and/or `patient:identifier`=`9000000009` and `_include` = `test`                                             | HTTP Status 422 and INVALID_PARAMETER error response                                                                  |        | Missing identifier                                                                | `patient:identifier`=`9000000009`                                                                                                                | HTTP Status 400 and MISSING_IDENTIFIER_VALUE error response                                                       |
```

Comment:
There seems to be an extra column in this row `| Missing identifier   `
<img width="1142" alt="image" src="https://github.com/user-attachments/assets/d5027308-9d80-429d-85cd-47b50c86c01f" />